src/openrct2/actions/ClearAction.hpp

```diff
@@ -171,7 +172,7 @@ DEFINE_GAME_ACTION(ClearAction, GAME_COMMAND_CLEAR_SCENERY, GameActionResult)
                         if (_itemsToClear & CLEARABLE_ITEMS::SCENERY_SMALL)
                         {
                             auto removeSceneryAction = SmallSceneryRemoveAction(
-                                { x * 32, y * 32, tileElement->base_height * 8 },
+                                { tilePos.x, tilePos.y, tileElement->base_height * 8 },
```

Comment:
If I'm not mistaken, you can create a CoordsXYZ(D) from a CoordsXY. If that is indeed the case, please modify the code accordingly.

Reply:
They do, and I actually used this on the other PRs today haha
Dealt with them, thanks for the tip :)